### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by sai84** [novitae/sterraxcyl#11]
Hello! I have a error during the export of mutual followers. Attached the screenshot
The text:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/sterra-2.1.4-py3.9.egg/sterra/core.py", line 84, in main
    raise IndexError
IndexError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/sterra", line 33, in <module>
    sys.exit(load_entry_point('sterra==2.1.4', 'console_scripts', 'sterra')())
  File "/usr/local/lib/python3.9/dist-packages/sterra-2.1.4-py3.9.egg/sterra/core.py", line 131, in main
    _ins.tagram(
  File "/usr/local/lib/python3.9/dist-packages/sterra-2.1.4-py3.9.egg/sterra/_ins.py", line 340, in tagram
    mutuals = traction(ul=mutualList, sd=sssid, ai=ex, xp=ex, dl=dl).strt()
TypeError: strt() missing 1 required positional argument: 'rtn'

![2022-02-08_22-00-15](https://user-images.githubusercontent.com/30548792/153066284-b6c425ad-1fdb-4523-9573-4b961f17efdc.png)
.

**Comment by novitae** [comment]
Hi !
I just finished on the version 2.2 of Sterra. The version you are using is an architectural disaster, and I am sorry of that.
The 2.2 is completely rebuilt and should normally work without any bad surprises.

I will release it officially in few days, I am still testing few things, but if you want, [here](https://github.com/novitae/SterraDev) is the unofficial link to get it. It is stable, I am just fighting with my computer that does weird things since I changed of python version.

To install it:
```
git clone https://github.com/novitae/SterraDev
cd SterraDev
python setup.py install
```

Some things has changed, check the wiki of its repository, all is explained.
Put issue there if you have some, im closing this for now since the version will be updated very soon !
